spec/models/user_spec.rb

```diff
@@ -475,6 +475,22 @@ def assert_good(username)
       expect(User.username_available?('DonKey')).to eq(false)
       expect(User.username_available?('test')).to eq(false)
     end
+
+    it 'tests when a username is reserved by an expression with wildcard' do
```

Comment:
The description here doesn't sound right to me. Usually the `it` block describes what the expected result should be. In this case, I would go with something like:

```
context 'when reserved usernames contains an expression" do
  it "should return the right value when username is matched against the expresssion" do
  end
end
```